ct/proxmox-datacenter-manager.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/raw/main/misc/build.func)
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: CrazyWolf13
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: Proxmox Server Solution GmbH
+
+# App Default Values
+APP="proxmox-datacenter-manager"
+var_tags="backup"
```

Comment:
```suggestion
var_tags="server,datacenter"
```
It is not only backup. But mine are just a suggestion, maybe you can come up with somethig better.